change-beta/@azure-communication-react-db7d6273-de66-40ff-abe3-9e4fca3c5312.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
I think it should still be none

Reply:
> I think it should still be none

I think bug fixes should be patch?
None is for things that does not affect our production code at all.

     Prerelease - bump prerelease version
     Patch      - bug fixes; no API changes.
     Minor      - small feature; backwards compatible API changes.
     None       - this change does not affect the published package in any way.

Reply:
Mention feature is still work in progress and should be use 'none' type. As soon as we release it, we should use 'patch' type